test/TestUtils.hs

```diff
@@ -122,3 +126,16 @@ isPkgDbAt _ _ = False
 
 instance IsString ModuleName where
     fromString = mkModuleName
+
+-- | Bracketing function for test/data/options-cradle environment to install tagged in package-db-a
+bracketTagged :: IO a -> IO a
+bracketTagged m = do
+    withDirectory_ "test/data/options-cradle" $ do
+        void $ system "cabal install --global --package-db=package-db-a --prefix=$(pwd) tagged"
```

Comment:
Honestly I'd rather the tests didn't start installing stuff with cabal, is there a particular reason you need to install a package rather than just, say using one of the boot packages like `containers` or `bytestring` or something?